### PR TITLE
feat(validations): add Validations component

### PR DESCRIPTION
**Pull request by maximilianoforlenza**
feat(validations): add Validations component

### CODE REVIEW

File: src/components/Validations/index.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import React, {Fragment} from 'react';
+import PropTypes from 'prop-types';
+import {isArray, isEmpty, map} from 'lodash';
```
Comment (navarroaxel) [inline src/components/Validations/index.js]: castArray

File: src/components/DwellingEditor/index.js [hunk 2]
```diff
@@ -17,6 +17,7 @@ class DwellingEditor extends Component {
         requestUpdateDwelling: PropTypes.func.isRequired,
         onPrevious: PropTypes.func.isRequired,
         onSubmit: PropTypes.func.isRequired,
+        validations: PropTypes.func,
```
Comment (navarroaxel) [inline src/components/DwellingEditor/index.js]: rename `validator`

File: src/components/Validations/index.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    <Fragment>
+        {map(
+            !isEmpty(validations) && !isArray(validations) ? [validations] : validations,
+            validation => <Alert key={validation.message}>{validation.message}</Alert>
+        )}
+    </Fragment>
+);
+
+Validations.propTypes = {
+    validations: PropTypes.arrayOf(PropTypes.shape({
```
Comment (navarroaxel) [inline src/components/Validations/index.js]: `validationResults`

File: src/components/Validations/index.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
+import React, {Fragment} from 'react';
+import PropTypes from 'prop-types';
+import {isArray, isEmpty, map} from 'lodash';
+import {Alert} from '@indec/react-native-commons';
+
+const Validations = ({validations}) => (
+    <Fragment>
+        {map(
+            !isEmpty(validations) && !isArray(validations) ? [validations] : validations,
+            validation => <Alert key={validation.message}>{validation.message}</Alert>
```
Comment (navarroaxel) [inline src/components/Validations/index.js]: rename `validationResult` or `result`

Comment (navarroaxel) [reply src/components/Validations/index.js]: `<Alert danger={result.type === 'danger'}` or `<Alert danger={result.isError} success={result.success}`

**Comment by maximilianoforlenza** [comment]
listo

**Pull request merged**